### ISSUE TITLE
export single system source install and cluster source install docs

### ISSUE THREAD

**Issue by yaobinshi** [fossology/fossology#60]
---

Author Name: **Mark Donohoe** 
Original Redmine Issue: 1371, http://www.fossology.org/issues/1371
Original Date: 2012/03/02
Original Assignee: Mary Laser

---

This defect is a reminder to export as text the single system and cluster system setups.  Then the cleaned up text files get checked into the svn as the install docs.  That way the wiki and text docs match and can be kept up to date easier.

**Comment by yaobinshi** [comment]
---

Original Redmine Comment
Author Name: **Mary Laser** 
Original Date: 2012/03/08 22:15:03 +0000

---

moving from Bug tracker to Documentation tracker

**Comment by yaobinshi** [comment]
---

Original Redmine Comment
Author Name: **Mary Laser** 
Original Date: 2012/04/02 23:14:15 +0000

---

INSTALL doc describing single system install from source is complete.  Cluster instructions will be deferred til after 2.0

**Comment by yaobinshi** [comment]
---

Original Redmine Comment
Author Name: **Mary Laser** 
Original Date: 2012/06/11 20:17:40 +0000

---

Need to define a global solution to maintain document consistency between wiki pages and informational text files maintained in subversion (i.e. INSTALL, README, Release_notes, etc.). This need not be done this iteration (2) but, should be done before the next release.  Will defer to a future iteration.

**Comment by yaobinshi** [comment]
---

Original Redmine Comment
Author Name: **Bob Gobeille** 
Original Date: 2012/10/08 18:35:12 +0000

---

Change unranked issues to rank 99 so they show up in a ranking report at the bottom.  Otherwise, they show up on top.

**Comment by mcjaeger** [comment]
closing because of missing pickup
